### PR TITLE
fix the upload profile image button

### LINKED ISSUE #613: Fixing the user profile picture update button
### Issue Description
Fixing the user profile picture update button

### Expected Behavior
image
Instead of edit text can add edit icon

### Current Behavior
image

### Screenshots
image

### Reproducibility

 - [ ] This issue is reproducible
 - [ ] This issue is not reproducible

### Severity/Priority
 - [ ] Critical
 - [ ] High
 - [ ] Medium
 - [x] Low

### Checklist

- [x] I have read and followed the project's code of conduct.
- [x] I have searched for similar issues before creating this one.
- [x] I have provided all the necessary information to understand and reproduce the issue.
- [x] I am willing to contribute to the resolution of this issue.

### PR DESCRIPTION
**Pull request by mdansarijaved**
Date: 7 Oct 2024

Developer Name: @mdansarijaved 

---

## Issue Ticket Number
- closes #613

## Description

This PR fixes the existing profile picture update button which was earlier not aligned.

### Documentation Updated?

- [ ] Yes
- [x] No

### Under Feature Flag

- [x] Yes
- [ ] No

### Database Changes

- [ ] Yes
- [x] No

### Breaking Changes

- [ ] Yes
- [x] No

### Development Tested?

- [x] Yes
- [ ] No

## Screenshots
<details>
<summary>Screenshot 1</summary>

image
</details>

## Test Coverage
<details>
<summary>Screenshot 1</summary>

image

</details>

### CODE REVIEW

File: app/styles/profile.css [hunk 1]
```diff
@@ -101,6 +102,20 @@
...
   padding: 0;
+  margin-top: 0.5rem;
+  color: var(--profile-edit-btn-clr);
+  font-weight: 600;
+  font-size: 1rem;
+}
+
+.profile-edit-button {
+  padding-inline: 5px;
+  padding-block: 3px;
```
Comment (yesyash) [inline app/styles/profile.css:113]: - these seem random to me can we use multiples of 4 (4, 8, 12...) as we follow it in our design system?

File: tests/unit/routes/profile-test.js
Comment (vinit717) [inline tests/unit/routes/profile-test.js:1]: Why is there no integration test for this?

**Review by vinit717 — CHANGES_REQUESTED** [review]

File: tests/unit/routes/profile-test.js
Comment (mdansarijaved) [reply tests/unit/routes/profile-test.js:1]: Test for this component already exists from what I can understand I've written test for dev feature flag for the changes I made.

File: app/styles/profile.css (hunk 1 above)
Comment (mdansarijaved) [reply app/styles/profile.css]: Hi @yesyash I've fixed this and padding is now in multiple of 4.

File: tests/unit/routes/profile-test.js
Comment (vinit717) [reply tests/unit/routes/profile-test.js:1]: Yes is possible to try to write a test for the dev flag also by this you will test the code you have changed behind feature flag

Comment (mdansarijaved) [reply tests/unit/routes/profile-test.js:1]: @vinit717 hy I've added some integration test as you suggested.

**Review by yesyash — APPROVED** [review]

File: tests/integration/components/profile-test.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+import { render, settled } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+
+module('Integration | Component | profile-component', function (hooks) {
+  setupRenderingTest(hooks);
+
+  test('button appearance based on isDev property', async function (assert) {
+    this.set('handleShowEditProfilePictureModal', function () {
+      //
+    });
```
Comment (vinit717) [inline tests/integration/components/profile-test.js:12]: Why this is empty 
If this is not necessary can you  remove this

Comment (mdansarijaved) [reply tests/integration/components/profile-test.js]: @vinit717  hy I was also testing the handleShowEditProfilePictureModal function. as you mentioned it should not be empty I've made changes accordingly. please review it thanks.

File: app/styles/profile.css [hunk 3]
```diff
@@ -101,6 +102,19 @@
...
+  font-size: 1rem;
+}
+
+.profile-edit-button {
+  padding: 8px;
+  z-index: 10;
+  border-radius: 100%;
+  position: absolute;
+  top: 8px;
+  right: 12px;
```
Comment (vinit717) [inline app/styles/profile.css:117]: why some places it is rem and other px?

Comment (mdansarijaved) [reply app/styles/profile.css]: We have already used px and rem both so used it. although i have changed px to rem.

**Review by vinit717 — APPROVED** [review]

**Review by yesyash — APPROVED** [review]

**Review by vinayak-trivedi — APPROVED** [review]

**Pull request merged**